src/test/java/org/folio/marccat/resources/AuthorityRecordTest.java

```diff
@@ -42,13 +48,26 @@ public void getDocumentCountById() throws IOException {
     String templateJson = IOUtils.toString(this.getClass().getResourceAsStream("/authority/name.json"),
```

Comment:
Looks like you do this several times... it might be worth creating method for this to reduce code duplication.

Reply:
I have changed it. Thanks